### PR TITLE
Update outdated part of the elastica.io documents. Storing and Indexing Page.

### PR DESCRIPTION
**Pull request by Numkil**
Update storing and indexing documents example to correctly reflect changes in elasticsearch 2.*

remove boost and analyzer options.

### CODE REVIEW

File: source/getting-started/storing-and-indexing-documents.markdown [hunk 1]
```diff
@@ -25,7 +25,7 @@ In elasticsearch, when you create an index, you define the number of shards and
...
 
-Let's create an index called twitter! We'll include two analyzers. Let's call them <code>indexAnalyzer</code> and <code>searchAnalyzer</code>, but you can use any name. <code>indexAnalyzer</code> defines how the data will be analyzed when it's indexed and <code>searchAnalyzer</code> defines how elasticsearch will analyze the search query. We'll reference these when we create the Mapping below. In this example we'll also use a custom snowball filter for the data.
+Let's create an index called twitter! We'll include two analyzers. .The indexes names are IMPORTANT because they decide when the analyzer will be used. The analyzer named "default_index" will be the analyzer used at index-time. The analyzer named "default_search" will be used when searching, if a custom analyzer is not provided in the query. <code>default_index</code> defines how the data will be analyzed when it's indexed and <code>default_search</code> defines how elasticsearch will analyze the search query. You can create analyzers with a random name, you can use these by referencing them in your query as the analyzer to use. In this example we'll also use a custom snowball filter for the data.
```
Comment (ruflin) [inline source/getting-started/storing-and-indexing-documents.markdown]: s/.The indexes/The indexes/

**Comment by ruflin** [comment]
LGTM. One minor typo.

I initially tried to have all code on Elastica.io also in the test suite to make sure it keeps working, but it seems like this got out of sync.

Thanks a lot for the contribution.

**Comment by Numkil** [comment]
No problem. Elastica made my life a lot easier on several occasions. :)

**Comment by ruflin** [comment]
Merged. The new version should now be available here: http://elastica.io/getting-started/storing-and-indexing-documents.html

**Pull request merged**